src/mame/konami/zr107.cpp

```diff
@@ -394,7 +394,7 @@ void zr107_state::sysreg_w(offs_t offset, uint8_t data)
 			break;
 
 		case 2: // Parallel data register
-			osd_printf_debug("Parallel data = %02X\n", data);
+			logerror("Parallel data = %02X\n", data);
```

Comment:
How frequently will this logging happen?  `osd_printf_debug` does nothing in non-debug builds, while `logerror` still sends the output to the debugger, log file and/or console, so it may have a performance impact.  Would it be better to put these behind `logmacro.h` conditional logging?